go/vt/sidecardb/dbname/name.go

```diff
@@ -0,0 +1,36 @@
+package dbname
+
+import (
+	"sync/atomic"
+
+	"vitess.io/vitess/go/vt/sqlparser"
```

Comment:
Perhaps this is the reason you chose to put this package where it is.
The way to break this up is that in `go/vt/constants`, you define only the `DefaultSidecarDBName`. The rest of this code can continue to live in `go/vt/sidecardb`.
This dependency is in fact another argument for creating a `constants` package. Otherwise any package that needs to know the sidecar db name will also start pulling in the sqlparser dependency whether it really needs it or not.

Reply:
Done